samples/to_dataframe/noxfile.py

```diff
@@ -82,7 +82,7 @@ def get_pytest_env_vars() -> Dict[str, str]:
 
 # DO NOT EDIT - automatically generated.
 # All versions used to tested samples.
-ALL_VERSIONS = ["2.7", "3.6", "3.7", "3.8"]
+ALL_VERSIONS = ["2.7", "3.6", "3.7", "3.8", "3.9"]
```

Comment:
I think this will likely result in test failures due to missing pyarrow dependencies.